test/suites/basic.sh

```diff
@@ -77,7 +77,7 @@ test_basic_usage() {
   lxc profile create priv
   lxc profile set priv security.privileged true
   lxc init testimage barpriv -p default -p priv
-  lxc publish barpriv --alias=foo-image prop1=val1
```

Comment:
Does this change imply that you have to pass all `--args` before positional arguments? I always think this is annoying when I have to order things :(